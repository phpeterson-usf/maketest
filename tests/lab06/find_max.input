$(repo_dir)/find_max_main
