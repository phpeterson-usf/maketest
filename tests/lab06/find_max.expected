find_max_c({1, 2, 5, 4, 3}, 5) = 5
find_max_s({1, 2, 5, 4, 3}, 5) = 5
find_max_c({-300, -200, -2, -1, 0, 100, 1, 2, 5, 4}, 10) = 100
find_max_s({-300, -200, -2, -1, 0, 100, 1, 2, 5, 4}, 10) = 100
